milatools/cli/commands.py

```diff
@@ -563,36 +606,89 @@ def code(
         alloc: Extra options to pass to slurm
     """
     here = Local()
-    remote = Remote("mila")
+    remote = Remote(cluster)
+
+    if cluster != "mila" and job is None and node is None:
+        if not any("--account" in flag for flag in alloc):
+            warnings.warn(
+                T.orange(
+                    "Warning: When using the DRAC clusters, you usually need to "
+                    "specify the account to use when submitting a job. You can specify "
+                    "this in the job resources with `--alloc`, like so: "
+                    "`--alloc --account=<account_to_use>`, for example:\n"
+                    f"mila code {path} --cluster {cluster} --alloc "
+                    f"--account=your-account-here"
+                )
```

Comment:
I think we could `ls ~/projects` dir to list the possible accounts that can be used (even if inactive accounts would be listed that could still add a minimum of validation) but this would not really explain the resources allocation so maybe we could instead suggest to visit the drac website? https://ccdb.alliancecan.ca/

Reply:
There's also an `sacctmgr` that we could use to list out the accounts. I'm actually using it here https://github.com/mila-iqia/milatools/pull/84/files#diff-11c81dbea84468d8c83ef30e30ecf1c2b4ee5cb470cddd2bbe59485508f3270aR93-R126 in #84

Reply:
I think it's a good idea to print a warning for now, and expect the users to know explicitly which account they want to use.